package/MDAnalysis/lib/mdamath.py

```diff
@@ -104,6 +104,7 @@ def angle(a, b):
     if x > 1.0:
         return 0.0
     elif x < -1.0:
+        x = np.clip(x,-1.0,-1.0)      
```

Comment:
@shfrz if I think @richardjgowers' point here was that the `if`/`else` construct should just go away if we use `np.clip`. The way you have now changed the code is such that this `np.clip` call will have no effect on solving issue #2632.

Reply:
@IAlibay , if I am getting it right  `x==-1.0001` would become `-1.0` which works fine?

Thanks!

Reply:
@shfrz Unless I'm missing something, by the time you've entered this part of the `if`/`else` construct, either way you'll return `-np.pi`?

Reply:
@IAlibay , so the right approach would be to remove the `if`/`else` block and set `x = np.clip(x,-1.0,1.0)?

Reply:
Yes, at least if we are going for a clip option, I believe that was @richardjgowers' original suggestion.